server/events/event_parser.go

```diff
@@ -789,13 +789,13 @@ func (e *EventParser) ParseAzureDevopsPull(pull *azuredevops.GitPullRequest) (pu
 
 	pullModel = models.PullRequest{
 		Author:     authorUsername,
-		HeadBranch: path.Base(headBranch),
+		HeadBranch: strings.SplitAfterN(headBranch, "/", 3)[2],
```

Comment:
Can you add a comment here that shows the two different types

Reply:
Not sure how this is working now.
Previously `"refs/heads/owner/feature/sourceBranch"` => `sourceBranch`.
Now `"refs/heads/owner/feature/sourceBranch"` => `owner/sourceBranch`.

Reply:
Ack!  Thanks.  I pushed too fast.

Reply:
That's what I get for using the exact same functions in the test as I do in the prod code.  I changed the test to now say:

```
HeadBranch: "feature/sourceBranch",
BaseBranch: "targetBranch",
```
The ADPull object I created in `server/events/vcs/fixtures/fixtures.go` was a bit suss and had [this](https://github.com/runatlantis/atlantis/blob/master/server/events/vcs/fixtures/fixtures.go#L83) listed as the git ref incorrectly, making the test fail:

`SourceRefName: azuredevops.String("refs/heads/owner/feature/sourceBranch"),`

Reply:
I also changed SplitAfterN() to Replace(). It's a little cleaner and should safely support both the git ref style "refs/heads/<branch>" as well as "<branch>" without having to check slice length in case it were to change unexpectedly.